ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -399,7 +402,14 @@ impl Mutation {
         ))
     }
 
-    async fn create_repository(ctx: &Context, name: String, git_url: String) -> Result<ID> {
+    async fn create_repository(
+        ctx: &Context,
+        name: String,
+        git_url: String,
+    ) -> Result<ID, RepositoryError> {
+        if !validate_identifier(&name) {
```

Comment:
Create a structure like [RegisterInput](https://github.com/TabbyML/tabby/blob/8c5b864ee7e64dc2a823d1bc505b2989920f5407/ee/tabby-webserver/src/service/auth.rs#L31), and use validator.